config/stack/manifests/resources/gkecluster.icon.svg

```diff
@@ -0,0 +1,10 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128">
```

Comment:
Are we sure we won't face some trademark issue using their icon?

Reply:
We are using their official icons that they bless for reuse by anyone in blog posts, whitepapers, datasheets, posters, etc.  So I think we're doing the right thing here.

Reply:
The icon pack's readme says this:
```
The Products logos contained in this icon library may be used freely and without permission to accurately reference Google's technology and tools, for instance in books or architecture diagrams.
```